.github/workflows/ci-aot.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish AOTCompatibility testApp
+
+on:
+  push:
+    branches: [ 'main*' ]
+    paths:
+    - 'src/OpenTelemetry.Instrumentation.Runtime/**'
+  pull_request:
+    branches: [ 'main*' ]
+    paths:
+    - 'src/OpenTelemetry.Instrumentation.Runtime/**'
```

Comment:
Exclude the path for markdown file.

You want to avoid running this check for PRs where only the markdown file of the project is updated.